README.md

```diff
@@ -6,8 +6,8 @@ Sandbox UI for the smartevents application.
 
 ## Prerequisites
 
-- Node 16
-- npm 7
+- Node 16.15.1
+- npm 8.11.0
```

Comment:
Beware of using LTS version of node and related npm version. The previous npm version has a bug with hoisting transitive deps that prevents the installation of the uniforms dependency.